packages/testing/package.json

```diff
@@ -13,6 +13,8 @@
   },
   "scripts": {
     "prebuild": "rm -rf lib",
+    "test:ci": "echo ''",
+    "test:dev": "echo ''",
```

Comment:
I think we can remove them.

```suggestion
```

Reply:
I defined in nx that the build task is a required for running the test task. If we remove this `npm run test:*` will fail because this package won't build.

Reply:
it should not be needed to execute `npm run test:*` we should update the readme that everything should be invoked with lerna by running:

```
npx lerna run test:dev --scope @netlify/build
```

or


```
npx lerna run build
```